lib/gcloud/bigquery/table.rb

```diff
@@ -127,6 +127,19 @@ def project_id
       end
 
       ##
+      # The Project ID, Dataset ID, and Table ID as a hash.
+      #
+      # :category: Attributes
+      #
+      def to_id_hash
```

Comment:
This should be hidden from the public API. I think this name could be more descriptive. What about `to_tbl_hash` or `to_tbl_ref` instead?